articles/azure-monitor/app/opencensus-python-dependency.md

```diff
@@ -162,11 +163,14 @@ from opencensus.trace import config_integration
 config_integration.trace_integrations(['sqlalchemy'])
 ```
 
+## Sample Applications
+
+You can find sample applications in the Azure Monitor OpenCensus Python samples repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/azure_monitor).
```

Comment:
I think just linking the django app under `Dependencies with "django" integration` is sufficient. We do not want to link to the generic samples folder.